Assets/Plugins/AndroidEditor/Editor/EOSOnPreprocessBuild_android.cs

```diff
@@ -221,4 +223,34 @@ public void ConfigureEOSDependentLibrary()
         }
     }
 
+    private void DetermineLibraryLinkingMethod()
+    {
+        var androidBuildConfigSection = EOSPluginEditorConfigEditor.GetConfigurationSectionEditor<EOSPluginEditorAndroidBuildConfigSection>();
+        androidBuildConfigSection?.Awake();
+        if (androidBuildConfigSection != null)
+        {
+            if (androidBuildConfigSection.GetCurrentConfig() == null)
+            {
+                androidBuildConfigSection.LoadConfigFromDisk();
+            }
+        }
+
+        var androidAssetFilepath = "../PlatformSpecificAssets/EOS/Android/";
+        var libSubpath = "/aar/eos-sdk.aar";
+
+        //string packagePathname = Path.GetFullPath("Packages/" + GetPackageName() + "/PlatformSpecificAssets~/" );
+        string dynamicPathName = Path.Combine(Application.dataPath, androidAssetFilepath, "dynamic-stdc++" + libSubpath);
```

Comment:
This still needs to have changes done so that it handles the case where it's running in the plugin, as the path names aren't shared. One can get the plugin path in the way described here https://github.com/PlayEveryWare/eos_plugin_for_unity/pull/178#discussion_r1046513278